content/blog/deploying-mongodb-application-on-hpe-greenlake-for-containers.md

```diff
@@ -0,0 +1,948 @@
+------
+
+------
+   Enable MongoDB's free cloud-based monitoring service, which will then receive and display
+   metrics about your deployment (disk utilization, CPU, operation statistics, etc).
+
+   The monitoring data will be available on a MongoDB website with a unique URL accessible to you
+   and anyone you share the URL with. MongoDB may use this information to make product
+   improvements and to suggest MongoDB products and deployment options to you.
+
+   To enable free monitoring, run the following command: db.enableFreeMonitoring()
+   To permanently disable this reminder, run the following command: db.disableFreeMonitoring()
+------
+
+test>
+
+Initialize Replicaset
+
+test> rs.initiate()
+{
+  info2: 'no configuration specified. Using a default configuration for the set',
+  me: 'mongo-0:27017',
+  ok: 1
+}
+rs0 [direct: other] test>
+
+Register mongo-0 pod as PRIMARY Replica
+
+rs0 [direct: other] test> var cfg = rs.conf();cfg.members[0].host="10.192.3.58:27017";rs.reconfig(cfg)
+{
+  ok: 1,
+  '$clusterTime': {
+    clusterTime: Timestamp({ t: 1669297890, i: 1 }),
+    signature: {
+      hash: Binary(Buffer.from("0000000000000000000000000000000000000000", "hex"), 0),
+      keyId: Long("0")
+    }
+  },
+  operationTime: Timestamp({ t: 1669297890, i: 1 })
+}
+
+Validate that mongo-0 pod is registered as Primary Replica
+
+rs0 [direct: other] test> rs.status()
+{
+  set: 'rs0',
+  date: ISODate("2022-11-24T13:52:34.600Z"),
+  myState: 1,
+  term: Long("1"),
+  syncSourceHost: '',
+  syncSourceId: -1,
+  heartbeatIntervalMillis: Long("2000"),
+  majorityVoteCount: 1,
+  writeMajorityCount: 1,
+  votingMembersCount: 1,
+  writableVotingMembersCount: 1,
+  optimes: {
+    lastCommittedOpTime: { ts: Timestamp({ t: 1669297950, i: 1 }), t: Long("1") },
+    lastCommittedWallTime: ISODate("2022-11-24T13:52:30.843Z"),
+    readConcernMajorityOpTime: { ts: Timestamp({ t: 1669297950, i: 1 }), t: Long("1") },
+    appliedOpTime: { ts: Timestamp({ t: 1669297950, i: 1 }), t: Long("1") },
+    durableOpTime: { ts: Timestamp({ t: 1669297950, i: 1 }), t: Long("1") },
+    lastAppliedWallTime: ISODate("2022-11-24T13:52:30.843Z"),
+    lastDurableWallTime: ISODate("2022-11-24T13:52:30.843Z")
+  },
+  lastStableRecoveryTimestamp: Timestamp({ t: 1669297900, i: 1 }),
+  electionCandidateMetrics: {
+    lastElectionReason: 'electionTimeout',
+    lastElectionDate: ISODate("2022-11-24T13:49:50.807Z"),
+    electionTerm: Long("1"),
+    lastCommittedOpTimeAtElection: { ts: Timestamp({ t: 1669297790, i: 1 }), t: Long("-1") },
+    lastSeenOpTimeAtElection: { ts: Timestamp({ t: 1669297790, i: 1 }), t: Long("-1") },
+    numVotesNeeded: 1,
+    priorityAtElection: 1,
+    electionTimeoutMillis: Long("10000"),
+    newTermStartDate: ISODate("2022-11-24T13:49:50.829Z"),
+    wMajorityWriteAvailabilityDate: ISODate("2022-11-24T13:49:50.840Z")
+  },
+  members: [
+    {
+      _id: 0,
+      name: '10.192.3.58:27017',
```

Comment:
Why is this highlighted in yellow in the preview?

Reply:
its out put of the cmd.